Small grammar fix for about_PSReadLine (#9424)
MicrosoftDocs/PowerShell-Docs: reference/7.2/PSReadLine/About/about_PSReadLine.md

$template -f $token
```

> The following commands are not being written to the history file:

Reviewer: ```suggestion
The following commands aren't being written to the history file:
```
